scripts/xtensa-build-zephyr.py

```diff
@@ -447,6 +448,10 @@ def build_platforms():
 			signing_key = default_rimage_key
 		sign_cmd += ["--tool-data", str(rimage_config), "--", "-k", str(signing_key)]
 
+		sof_tag = subprocess.check_output(["git", "describe", "--tags"]).strip().decode()
```

Comment:
This is really an independent feature, let's make it a function, for example, `extract_sof_version_from_git()` or simplely `extract_sof_version`, let's hear more from @marc-hb

Reply:
@aiChaoSONG Okay. Thank you!  I updated the PR to define a function get_sof_version() to get SOF FW file version string.